app/views/accounts/settings.html.erb

```diff
@@ -450,6 +450,37 @@ TEXT
           </fieldset>
         <% end %>
 
+        <% if @expose_account_level_first_day_start_time %>
+          <fieldset>
+            <legend><%= t(:course_start_time, 'Default Course Start Time') %>
+            </legend>
+              <table>
+            <tr>
+              <td class="form-label"><%= f.blabel :course_start_time, :en => "Starts" %></td>
+              <td class="formtable small-select">
+                <%= f.fields_for :settings do |settings| %>
+                  <%= settings.number_field :course_start_time_hour,
+                                            :value => @course_start_time_hour,
+                                            :default => @course_start_time_hour,
+                                            :min => "00",
```

Comment:
I think the minimum here is 1, 1-12 not 0-12

Reply:
@JamesWingert made these changes